lib/dfe_sign_in_api.rb

```diff
@@ -4,7 +4,7 @@ class ForbiddenRequestError < StandardError; end
   class UnknownResponseError < StandardError; end
 
   class API
-    USERS_PAGE_SIZE = 25
+    USERS_PAGE_SIZE = 100
```

Comment:
Did DSI say they weren't going to upgrade the limit beyond 100?

Reply:
I'm still waiting for their reply 🤔

Reply:
It's in their backlog now, but it is uncertain when this change going to happen